config/rbac/role.yaml

```diff
@@ -7,10 +7,20 @@ metadata:
   name: vault-secrets-operator
 rules:
 - apiGroups:
-  - coordination.k8s.io
   - ""
   resources:
-  - configmaps
+  - leases
```

Comment:
Shouldn't this be `configmaps` instead of `leases` like in the other file?

Reply:
Done! Thank you for catching this!